setup.py

```diff
@@ -1,5 +1,4 @@
 """Defines packaging and distribution."""
 from setuptools import setup, find_packages
 
-setup(packages=find_packages(),
-      zip_safe=False)
+setup(packages=find_packages())
```

Comment:
We can do this in setup.cfg

```suggestion
setup()
```